Tests/AsyncHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -531,7 +552,8 @@ internal final class HttpBinHandler: ChannelInboundHandler {
                 context.writeAndFlush(wrapOutboundOut(.head(HTTPResponseHead(version: HTTPVersion(major: 1, minor: 1), status: .ok))), promise: nil)
                 return
             case "/events/10/1": // TODO: parse path
-                context.write(wrapOutboundOut(.head(HTTPResponseHead(version: HTTPVersion(major: 1, minor: 1), status: .ok))), promise: nil)
+                let headers = HTTPHeaders([("Content-Length", "50")])
```

Comment:
This changes the behaviour of this endpoint (it previously automatically sent chunked encoding, now it doesn't). I don't _think_ any of the tests rely on that behaviour, but as it's fairly cheap to add new endpoints I think I'd rather we added a new one of these.

We can keep the common code by factoring this out into a method that conditionally adds content-length if we want.